app/src/main/assets/india.ics

```diff
@@ -2,323 +2,576 @@ BEGIN:VCALENDAR
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:First Day of Sharad Navratri
+UID:ind_47
+DTSTART;VALUE=DATE:20231015
+DTEND;VALUE=DATE:20231016
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:First Day of Durga Puja Festivities
+UID:ind_48
+DTSTART;VALUE=DATE:20231020
+DTEND;VALUE=DATE:20231021
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Maha Saptami
+UID:ind_49
+DTSTART;VALUE=DATE:20231021
+DTEND;VALUE=DATE:20231022
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Maha Ashtami
+UID:ind_50
+DTSTART;VALUE=DATE:20231022
+DTEND;VALUE=DATE:20231023
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Maha Navami
+UID:ind_51
+DTSTART;VALUE=DATE:20231023
+DTEND;VALUE=DATE:20231024
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Dussehra
+UID:ind_52
+DTSTART;VALUE=DATE:20231024
+DTEND;VALUE=DATE:20231025
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Maharishi Valmiki Jayanti
+UID:ind_53
+DTSTART;VALUE=DATE:20231028
+DTEND;VALUE=DATE:20231029
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Karaka Chathurthi(Karva Chauth)
+UID:ind_54
+DTSTART;VALUE=DATE:20231101
+DTEND;VALUE=DATE:20231102
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Diwali/Deepavali
+UID:ind_55
+DTSTART;VALUE=DATE:20231112
+DTEND;VALUE=DATE:20231113
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Naraka Chaturdasi
+UID:ind_56
+DTSTART;VALUE=DATE:20231112
+DTEND;VALUE=DATE:20231113
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Govardhan Puja
+UID:ind_57
+DTSTART;VALUE=DATE:20231113
+DTEND;VALUE=DATE:20231114
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Bhai Duj
+UID:ind_58
+DTSTART;VALUE=DATE:20231114
+DTEND;VALUE=DATE:20231115
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Nutan Varsh
+UID:ind_59
+DTSTART;VALUE=DATE:20231114
+DTEND;VALUE=DATE:20231115
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Chhat Puja (Pratihar Sashthi/Surya Sashthi)
+UID:ind_60
+DTSTART;VALUE=DATE:20231119
+DTEND;VALUE=DATE:20231120
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Guru Tegh Bahadur's Martyrdom Day
+UID:ind_61
+DTSTART;VALUE=DATE:20231124
+DTEND;VALUE=DATE:20231125
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Guru Nanak Jayanti
+UID:ind_62
+DTSTART;VALUE=DATE:20231127
+DTEND;VALUE=DATE:20231128
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
+SUMMARY:Christmas Eve
+UID:ind_63
+DTSTART;VALUE=DATE:20231224
+DTEND;VALUE=DATE:20231225
+STATUS:CONFIRMED
+RRULE:FREQ=YEARLY;INTERVAL=1
+END:VEVENT
+
+BEGIN:VEVENT
```

Comment:
remove all empty lines please

Reply:
Sorry, I thought they would make it easier to navigate for next time since it need to be updated on yearly basis.